### PR TITLE
Feat(web-twig): Introduce RadioField component

### PR DESCRIPTION
**Pull request by janicekt**
Snímek obrazovky 2022-07-29 v 12 56 16

### CODE REVIEW

File: packages/web-twig/src/Resources/components/RadioField/RadioField.twig [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+{%- set _classNames = [ _rootClassName, _rootDisabledClassName, _class ] -%}
+{%- set _labelClassName = [ _labelClassName, _labelHiddenClassName ] -%}
+{%- set _mainPropsWithoutId = props | filter((value, prop) => prop != 'id') -%}
+
+<label for="{{ _id }}" {{ mainProps(_mainPropsWithoutId) }} {{ classProp(_classNames) }}>
+    <input
+        {{ inputProps(props) }}
+        type="radio"
+        id="{{ _id }}"
+        name="{{ _name }}"
```
[the remark this answers is not in the document]
Comment (adamkudrna) [reply packages/web-twig/src/Resources/components/RadioField/RadioField.twig:33]: It's the same question for `CheckboxField`… 👀

**Review by adamkudrna — COMMENTED** [review]
Looks good! I only have questions about the optional attributes.

File: packages/web-twig/src/Resources/components/RadioField/RadioField.twig [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{# API #}
+{%- set _class = props.class | default(null) -%}
+{%- set _id = props.id | default(null) -%}
+{%- set _isChecked = props.isChecked | default(false) | boolprop -%}
+{%- set _isDisabled = props.isDisabled | default(false) | boolprop -%}
+{%- set _isLabelHidden = props.isLabelHidden | default(false) | boolprop -%}
+{%- set _label = props.label | default(null) -%}
```
[the remark this answers is not in the document]
Comment (janicekt) [reply packages/web-twig/src/Resources/components/RadioField/RadioField.twig:7]: I did it according to the checkboxes to maintain consistency.. but hopefully I can get to it next week..

File: packages/web-twig/src/Resources/components/RadioField/RadioField.twig (hunk 1 above)
[the remark this answers is not in the document]
Comment (janicekt) [reply packages/web-twig/src/Resources/components/RadioField/RadioField.twig:34]: fixed in RadioField ... ChecklistField is not in the scope of this issue...

File: packages/web-twig/src/Resources/components/RadioField/RadioField.twig (hunk 2 above)
[the remark this answers is not in the document]
Comment (janicekt) [reply packages/web-twig/src/Resources/components/RadioField/RadioField.twig]: I fixed label requirement status

File: packages/web-twig/src/Resources/components/RadioField/RadioField.twig (hunk 1 above)
[the remark this answers is not in the document]
Comment (adamkudrna) [reply packages/web-twig/src/Resources/components/RadioField/RadioField.twig:34]: Thanks!

> ChecklistField is not in the scope of this issue

Well, it's not in the scope, but we are used to do small bug fixes even out of the scope—as long as they are small 🙂.

**Review by adamkudrna — APPROVED** [review]

**Review by literat — APPROVED** [review]

**Pull request merged**